### PR TITLE
WIP, ENH: rev 1 request -- more training data

### PR DESCRIPTION
**Pull request by tylerjereddy**
* Related to gh-43; addressing reviewer comments for manuscript PCOMPBIOL-D-25-01998.

* This branch aims to address the reviewer 1 comment asking for us to investigate what happens when we use more of the now-curated data for training. The approach is quite hacky, and we do not expect the full testsuite to pass for the reasons described below. However, the workflow does run to completion on a high-memory `darwin` node as described at:
https://github.com/lanl/ldrd_virus_work/issues/43#issuecomment-3768921463

* The single-seed workflow run with this new data split had an average ROC AUC on test across the eight estimators of $0.77875 \pm 0.00927$. This predictive performance is quite close to the original (rebalanced/shuffled split) human prediction metrics reported across ten random seeds in Figure 1 (ROC AUC of $0.769 \pm 0.014$ and $0.784 \pm 0.013$ without and with hyperopt, respectively).

* `Relabeled_Train_Human_Shuffled.csv` now contains 80 % of the data (1268 records, up from original 849 records), while the testing data has been reduced in size with `Relabeled_Test_Human_Shuffled.csv` now containing only 317 records (down from 736 records). The method of transfer of data from test to training was not particularly disciplined-- the only constraint was that the test data must retain at least one negative response and one positive response value for ROC AUC calculation.

* Note that hyperparameters have not been optimized for this new data split, we are simply reusing the hyperparameters from the original human shuffled split. Also, and obviously, the appropriate way to do this long-term would be to use a separate set of CSV files for this new data split, not to override the human shuffled CSV files we still use for the workflow. However, because of limited time, this is a fast workaround for an assessment of ROC AUC for human infection with this new data split.

* The plotting of the "x" at the intersection between the ROC plot diagonal and the ROC curve proper inside
`plot_roc_curve_comparison()` has been suppressed because of the issue detailed at:
https://github.com/lanl/ldrd_virus_work/issues/43#issuecomment-3766375105 This is an undesirable temporary workaround, and may only be acceptable because the reviewer comment is not focused on the EER.

### CODE REVIEW

File: viral_seq/analysis/classifier.py [hunk 1]
```diff
@@ -642,14 +642,14 @@ def plot_roc_curve_comparison(
...
-                ms=12,
-            )
+            #ax.plot(
+                #intersection.x,  # type: ignore
+                #intersection.y,  # type: ignore
+                #marker="x",
+                #label=f"{name} {eer_threshold = :.2e}",
+                #alpha=1.0,
+                #ms=12,
+            #)
```
Comment (tylerjereddy) [inline viral_seq/analysis/classifier.py:652]: This is obviously not acceptable for merging -- it is temporary hack as we try to address reviewer comments, which do not require the EER to be investigated.

If we want to merge this eventually, the issue at https://github.com/lanl/ldrd_virus_work/issues/43#issuecomment-3766375105 needs to be investigated and resolved gracefully.

File: viral_seq/data/Relabeled_Train_Human_Shuffled.csv [hunk 2]
```diff
@@ -848,3 +848,422 @@
 846,Wutai mosquito phasivirus,NC_031317.1  NC_031318.1  NC_031319.1,False
 847,Bat associated cyclovirus 1,NC_034547.1,False
 848,Iriri curiovirus,NC_034544.1,False
+74,Hepacivirus B,NC_038426.1,True
```
Comment (tylerjereddy) [inline viral_seq/data/Relabeled_Train_Human_Shuffled.csv:851]: Rather than modifying these two "Human Shuffled" CSV files, which are still used for the main workflow, if we want to merge this the new data should be stored in separate CSV files to avoid interference with original "Human Shuffled" data split.

That would then, in turn, most likely we require code and command line adjustments to handle the new data set.

Since time is incredibly tight (haven't had team/trainee support in months...), this temporary hack is being used to attempt to address manuscript reviewer comments for now...
